### ISSUE TITLE
Incorrect usage of updateContextEntry

### ISSUE THREAD

**Issue by reilem** [felangel/fresh#42]
Currently this library uses `updateContextEntry` incorrectly in the latest version of flutter_graphql. The function `updateContextEntry` now no longer mutates the request object, but is a pure function. Now it returns the new request that you need to pass to the forward function.
Essentially this bit of the code:
```dart
request.updateContextEntry<HttpLinkHeaders>(
    (headers) => HttpLinkHeaders(
      headers: {
        ...headers?.headers ?? <String, String>{},
      }..addAll(tokenHeaders),
    ),
  );
```
Does nothing. It should be:
```dart
final newRequest = request.updateContextEntry<HttpLinkHeaders>(
    (headers) => HttpLinkHeaders(
      headers: {
        ...headers?.headers ?? <String, String>{},
      }..addAll(tokenHeaders),
    ),
  );
```
And the you need to work with newRequest down the line. The result of this mistake makes it so that the authorization header is not put in the context and authentication fails.

**Comment by vytautas-pranskunas-** [comment]
watching

**Comment by felangel** [comment]
Hi @reilem 👋 
Thanks for opening an issue and for catching this!

It should be fixed as part of #43 and available as part of [v0.5.1](https://pub.dev/packages/fresh_graphql). Sorry for the inconvenience!

**Comment by vytautas-pranskunas-** [comment]
I think the same issue is with 0.4.0 version.
Do you think you can release 0.4.1? because I cannot migrate to null safety yet but headers are not sent...

**Comment by reilem** [comment]
> Hi @reilem 👋
> Thanks for opening an issue and for catching this!
> 
> It should be fixed as part of #43 and available as part of [v0.5.1](https://pub.dev/packages/fresh_graphql). Sorry for the inconvenience!

No worries, and thanks for the quick fix!

**Comment by vytautas-pranskunas-** [comment]
@felangel hi, can it be added to 0.4.1v?
